packages/dashmate/configs/defaults/getBaseConfigFactory.js

```diff
@@ -311,6 +311,11 @@ export default function getBaseConfigFactory(homeDir) {
               host: '127.0.0.1',
               port: 29090,
             },
+            grovedbVisualizer: {
+              enabled: true,
```

Comment:
Let's disable by default

Reply:
I thought we discussed this morning that we want it on testnet by default

Reply:
No, at least I did not understand that. We need it on testnet but it shouldn't be defaulting to enabled.

Reply:
Yes, but only for our nodes (not 3rd party). We will manage it with configuration in the deployment tool